drivers/bridge/bridge.go

```diff
@@ -876,6 +877,54 @@ func setHairpinMode(nlh *netlink.Handle, link netlink.Link, enable bool) error {
 	return nil
 }
 
+func waitForIfaces(ifaceNames []string, timeout time.Duration, trigger func() error) error {
+	// Map interface names to their readiness states (false by default)
+	ifaceStates := make(map[string]bool)
+	allRunning := func() bool {
+		for _, ifaceState := range ifaceStates {
+			if !ifaceState {
+				return false
+			}
+		}
+		return true
+	}
+	for _, ifaceName := range ifaceNames {
+		ifaceStates[ifaceName] = false
+	}
+
+	// Start watching for link changes
+	notifyLink := make(chan netlink.LinkUpdate)
+	notifyDone := make(chan struct{})
+	netlink.LinkSubscribe(notifyLink, notifyDone)
```

Comment:
The CI test still failed when specifying the namespace.  (I had to export the `getHandler` function to try it.)  Are you aware of anything else that might make it behave differently than a QEMU VM?  I would hope it's not actually taking longer than a second to set up the interfaces.